phonegap/hooks/overwrite_ios_view.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+// Adapted from:
+// http://devgirl.org/2013/11/12/three-hooks-your-cordovaphonegap-project-needs/
+
+var filestocopy = [{
+	'hooks/MainViewController.m':
+		'platforms/ios/Waterfronts/Classes/MainViewController.m'
```

Comment:
This file location only exists if the phonegap build runs against a project called "Waterfronts". We should definitely update this to whatever Oregon's Catch parses into.